README.md

```diff
@@ -93,13 +94,23 @@ Choose deployment option (recommended default: 1)
 
 ### ASP.NET Core web applications
 ASP.NET Core applications can be deployed either to virtual servers with AWS Elastic Beanstalk or containers with Amazon Elastic Container Service (Amazon ECS). If you wish to deploy your application as a container and your project does not yet have a `Dockerfile` one will be generated for you into your project during deployment.
+Using the deploy tool, you will be able to deploy ASP.NET Core web applications to AWS using the AWS Cloud Development Kit (CDK). 
+For users coming from the previous AWS Visual Studio Toolkit experience who have previous Elastic Beanstalk deployments and environments, you will be able to use the deploy tool to deploy to those environments.
+
+#### Elastic Beanstalk backwards compatibility support
+Currently, the AWS Visual Studio toolkit allows users to deploy their ASP.NET Core web applications to Elastic Beanstalk using a wizard that uses the AWS .NET SDK on the backend to perform the deployment. This process creates the necessary Elastic Beanstalk resources such as a Beanstalk Application and Beanstalk Environment using the AWS .NET SDK. 
+
+The new deploy tool uses AWS CDK to perform the deployments to AWS which create a CloudFormation stack that manages all the Elastic Beanstalk resources. This means that the deploy tool cannot update the existing Beanstalk resources that were created using the old AWS Visual Studio toolkit experience. In order to tackle this issue, the deploy tool now supports deploying to existing Elastic Beanstalk environments, in addition to CloudFormation stacks. 
+
+The deploy tool will now detect existing Elastic Beanstalk environments in your AWS account and list them alongside the existing CloudFormation stacks (that were created using the deploy tool). You will be able to select an existing Beanstalk environment and deploy your application that environment as well as updating some settings that were available in the old AWS Visual Studio toolkit experience. On the backend, the deploy tool is using the AWS .NET SDK to perform the deployment to existing Beanstalk Environments.
+
+Note: Deploying to existing Beanstalk Environments will not create a new CloudFormation stack and migrate the existing resources to be managed by CloudFormation. In order to do that, you will need to deploy your application to a new stack which will use the AWS CDK to manage all the resource in CloudFormation.
```

Comment:
_"and **will not** migrate the existing resource...."_

Reply:
Done